examples/src/main/java/org/apache/spark/examples/ml/JavaOneVsRestExample.java

```diff
@@ -0,0 +1,225 @@
+
+  private static class Params {
+    String input;
+    String testInput = null;
+    Integer maxIter = 100;
+    double tol = 1E-6;
+    boolean fitIntercept = true;
+    Double regParam = null;
+    Double elasticNetParam = null;
+    double fracTest = 0.2;
+  }
+
+  public static void main(String[] args) {
+    // parse the arguments
+    Params params = parse(args);
+    SparkConf conf = new SparkConf().setAppName("JavaOneVsRestExample");
+    JavaSparkContext jsc = new JavaSparkContext(conf);
+    SQLContext jsql = new SQLContext(jsc);
+
+    // configure the base classifier
+    LogisticRegression classifier = new LogisticRegression()
+      .setMaxIter(params.maxIter)
+      .setTol(params.tol)
+      .setFitIntercept(params.fitIntercept);
+
+    if (params.regParam != null) {
+      classifier.setRegParam(params.regParam);
+    }
+    if (params.elasticNetParam != null) {
+      classifier.setElasticNetParam(params.elasticNetParam);
+    }
+
+    // instantiate the One Vs Rest Classifier
+    OneVsRest ova = new OneVsRest();
+    ova.setClassifier(classifier);
+
+    String input = params.input;
+    RDD<LabeledPoint> inputData = MLUtils.loadLibSVMFile(jsc.sc(), input);
+    RDD<LabeledPoint> train;
+    RDD<LabeledPoint> test;
+    String testInput = params.testInput;
+
+    // compute the train/test split: if testInput is not provided use part of input.
+    if (testInput != null) {
+      train = inputData;
+      test = MLUtils.loadLibSVMFile(jsc.sc(), testInput);
+    } else {
+      double f = params.fracTest;
+      RDD<LabeledPoint>[] tmp = inputData.randomSplit(new double[]{1 - f, f}, 12345);
+      train = tmp[0];
+      test = tmp[1];
+    }
+
+    // train the multiclass model.
+    DataFrame trainingDataframe = jsql.createDataFrame(train, LabeledPoint.class);
+    OneVsRestModel ovaModel = ova.fit(trainingDataframe.cache());
+
+    // score the model on test data.
+    DataFrame testDataframe = jsql.createDataFrame(test, LabeledPoint.class);
+    DataFrame predictions = ovaModel.transform(testDataframe.cache())
+      .select("prediction", "label");
+
+    MulticlassMetrics metrics = new MulticlassMetrics(predictions);
+
+    // output the confusion matrix.
+    System.out.println("ConfusionMatrix");
+    System.out.println(metrics.confusionMatrix().toString());
+
+    StructField predictionColSchema = predictions.schema().apply("prediction");
+    Integer numClasses = (Integer) MetadataUtils.getNumClasses(predictionColSchema).get();
+
+    // compute the false positive rate per label
+    StringBuilder results = new StringBuilder();
+    results.append("label\tfpr\n");
+    for (int label = 0; label < numClasses; label++) {
+      results.append(label);
+      results.append("\t");
+      results.append(metrics.falsePositiveRate((double) label));
+      results.append("\n");
+    }
+    System.out.println(results);
+
+    jsc.stop();
+  }
+
+  private static Params parse(String[] args) {
+    String input = args[0];
```

Comment:
Could you please check for args being empty and print the help message if it is?